docs/direct-access-urls.md

```diff
@@ -231,19 +254,19 @@ ___
 
 ## Performance and scalability
 Each call to one of the new `.../request-direct-access-url` Alfresco REST endpoints results in the
-creation of a new (and separate) pre-signed URL for an AWS S3 object.
+creation of a new (and separate) pre-signed URL for an AWS S3 / Azure Blob object.
 
-The generation of a pre-signed URL is a purely AWS SDK **client-side operation** - meaning the
+The generation of a pre-signed URL is a purely AWS/Azure (TBC) SDK **client-side operation** - meaning the
```

Comment:
Do we need TBC (we'll need to remember to come back and remove it) ? Can we check via experimental impl ?

Reply:
Checked - no TBC needed.